src/js/editor/editor.js

```diff
@@ -571,8 +520,9 @@ class Editor {
     section.renderNode.markDirty();
   }
 
-  reparseSection(section) {
-    this._parser.reparseSection(section, this._renderTree);
+  _reparseCurentSection() {
+    const {headSection:currentSection } = this.cursor.offsets;
```

Comment:
oh — in case it isn't obvious, this method is on the editor, not the post parser. The post parser itself has no knowledge of cursors.